src/SigasiProjectCreator/Creator.py

```diff
@@ -309,13 +309,18 @@ def write(self, destination):
             os.makedirs(settings_dir)
         prefs_file_path = "com.sigasi.hdt.{0}.{1}.prefs".format(self.lang, str(self.lang).title())
         prefs_file = os.path.join(settings_dir, prefs_file_path)
+        # TODO why not overwrite?
         if not os.path.exists(prefs_file):
             rel_verilog_includes = []
             abs_destination = absnormpath(destination)
+            # TODO improve PATH handling
             for path in self.verilog_includes:
-                abs_path = absnormpath(path)
-                relative_path = os.path.relpath(path, abs_destination)
-                rel_verilog_includes.append(posixpath(relative_path))
+                if str(path).startswith('Common Libraries'):
+                    rel_verilog_includes.append(path)
+                else:
+                    # TODO why is this line here? # abs_path = absnormpath(path)
```

Comment:
This would resolve symlinks and make the path absolute. `os.path.relpath` might not work with relative paths or paths containing symlinks.

Reply:
Nah, I mean, `abs_path` (the result of this line of code) is never used. Message left for later cleanup.